### ISSUE TITLE
Death Stranding Director's Cut

### ISSUE THREAD

**Issue by TheAlpha31** [emoose/DLSSTweaks#63]
Just wanted to report that forcing DLAA and the other overrides seems to be working well in Death Stranding Director's Cut. 

Using DLSSTweaks ver. [0.200.5-beta4](https://github.com/emoose/DLSSTweaks/releases/tag/0.200.5-beta4), DLSS ver. 3.1.11, game ver. 1.002. 

Available DLSS modes are Quality, Performance, and Ultra Performance. 

Quality and Performance modes default to preset C, while Ultra Performance defaults to preset B. Forcing preset D causes some obvious ghosting around particle effects, which can be particularly bad in situations where particles pass between the player character and camera. 

![DLSSPreset C](https://user-images.githubusercontent.com/50899632/229658235-38fecd9a-aa19-452b-ba01-5411b697e6e5.jpg)
Default preset C

![DLSSPreset D](https://user-images.githubusercontent.com/50899632/229658266-221ee6f2-e37f-4e8d-a805-ef22470f6fc5.jpg)
Force preset D, notice the ghosting below the rising black particle effects.

In older versions of DLSSTweaks, `OverrideAppId` needs to be set to true before altering `DLSSPresets`. Should not be needed after DLSSTweaks ver. 0.200.5. 

Logging indicates custom exposure values are not being used and AutoExposure is disabled by default, so `OverrideAutoExposure` should be enabled (set to 1). Improves image quality some; haven't seen any drawbacks.

**Comment by TheSpider12** [comment]
> Just wanted to report that forcing DLAA and the other overrides seems to be working well in Death Stranding Director's Cut.
> 
> Using DLSSTweaks ver. [0.200.5-beta4](https://github.com/emoose/DLSSTweaks/releases/tag/0.200.5-beta4), DLSS ver. 3.1.11, game ver. 1.002.
> 
> Available DLSS modes are Quality, Performance, and Ultra Performance.
> 
> Quality and Performance modes default to preset C, while Ultra Performance defaults to preset B. I don't recommend forcing preset D, it causes some obvious ghosting around particle effects which can be particularly bad in situations where particles pass between the player character and camera. Preset F seems okay if you want that ultra-sharp look.
> 
> ![DLSSPreset C](https://user-images.githubusercontent.com/50899632/229658235-38fecd9a-aa19-452b-ba01-5411b697e6e5.jpg) Default preset C
> 
> ![DLSSPreset D](https://user-images.githubusercontent.com/50899632/229658266-221ee6f2-e37f-4e8d-a805-ef22470f6fc5.jpg) Force preset D, notice the ghosting below the rising black particles.
> 
> In older versions of DLSSTweaks, `OverrideAppId` needs to be set to true before altering `DLSSPresets`. Should not be needed after DLSSTweaks ver. 0.200.5.
> 
> Logging indicates custom exposure values are not being used and AutoExposure is disabled by default, so `OverrideAutoExposure` should be enabled (set to 1). Improves image quality some; haven't seen any drawbacks.

Per my experience with presets in several games, preset F is the blurriest, and preset D is the sharpest, so I'm a bit surprised to read your sentence "Preset F seems okay if you want that ultra-sharp look".

Have you tried DLDSR 2.5x + DLSS Quality in games? Somehow forced DLAA is blurrier than DLDSR 2.5x + DLSS Quality, even though they are supposed to both "arrive" at native res.

**Comment by TheAlpha31** [comment]
> Per my experience with presets in several games, preset F is the blurriest, and preset D is the sharpest, so I'm a bit surprised to read your sentence "Preset F seems okay if you want that ultra-sharp look".

You're right, I think I said it backwards then what I meant. There seemed to be very little ghosting compared to D and a bit less then C, but it introduced some rough line edges. Maybe an 'aliased' image is more accurate? Anyhow, I'm preferring preset C on DLSS Quality. 

> Have you tried DLDSR 2.5x + DLSS Quality in games? Somehow forced DLAA is blurrier than DLDSR 2.5x + DLSS Quality, even though they are supposed to both "arrive" at native res.

No, I haven't tried mixing DLDSR and DLSS yet. 

As I understand it, the smoothness slider in the control panel is kind of operating backwards with DLDSR compared to regular DSR. With DSR it adds more smoothing going from 0% to 100%, but with DLDSR it adds more sharpening going from 100% to 0%, if that makes sense. So with DLDSR, if you have the slider at 33%, it's actually doing something like 77% sharpening in the background. 

I would be curious to see if just adding a sharpening filter through the control panel would give you the same kind of picture you're looking for. I personally don't really like sharpening filters, so I haven't messed with it too much.

**Comment by TheAlpha31** [comment]
Oh yeah, I uploaded some still DLAA screenshots at [https://imgsli.com/MTY2OTA4](https://imgsli.com/MTY2OTA4). Presets C and D look pretty similar, but F is a good bit blurrier/softer, like you said. 

Here are some DLAA screenshots with particles in motion: [https://imgsli.com/MTY2OTA5](https://imgsli.com/MTY2OTA5). Pretty similar as before, but the ghosting trails pretty badly on Preset D. 

For good measure, here's some DLSS Quality screenshots in the same spot: [https://imgsli.com/MTY2OTEw](https://imgsli.com/MTY2OTEw). I guess 'aliasing' isn't really right either, but I'm finding things in motion look a bit strange with preset F. Maybe shimmery? I'm not sure of the right word.

**Comment by TheSpider12** [comment]
> >As I understand it, the smoothness slider in the control panel is kind of operating backwards with DLDSR compared to regular DSR. With DSR it adds more smoothing going from 0% to 100%, but with DLDSR it adds more sharpening going from 100% to 0%, if that makes sense. So with DLDSR, if you have the slider at 33%, it's actually doing something like 77% sharpening in the background.
> 
> I would be curious to see if just adding a sharpening filter through the control panel would give you the same kind of picture you're looking for. I personally don't really like sharpening filters, so I haven't messed with it too much.

I leave the smoothness slider at 80%. To my eyes it doesn't look over-sharpened like 33% or blurry like 100%.

**Comment by TheAlpha31** [comment]
> Maybe you can try DLDSR 2.25x + DLSS Quality combo on Death Stranding and compare the image quality as well as frame rate to DLAA. 

It is coming out a bit sharper, but I'm also dropping from 90 FPS down to ~80 fps. No sharpening on DLAA image, 100% smoothness on DLDSR image (though I'm not sure if smoothness would even show up on a screenshot): [https://imgsli.com/MTY3MDA0](https://imgsli.com/MTY3MDA0) 

> In theory they should be the same but I'm seeing discrepancies in the few games I tested.

I'm don't know about them necessarily being 'the same'. I'd think that, generally, DLDSR+DLSS should look better than just DLAA. There's more going on under the hood than just the number of pixels.

**Comment by TheSpider12** [comment]
> > Maybe you can try DLDSR 2.25x + DLSS Quality combo on Death Stranding and compare the image quality as well as frame rate to DLAA.
> 
> It is coming out a bit sharper, but I'm also dropping from 90 FPS down to ~80 fps. No sharpening on DLAA image, 100% smoothness on DLDSR image (though I'm not sure if smoothness would even show up on a screenshot): https://imgsli.com/MTY3MDA0
> 
> > In theory they should be the same but I'm seeing discrepancies in the few games I tested.
> 
> I don't know about them necessarily being 'the same'. I'd think that, generally, DLDSR+DLSS should look better than just DLAA. There's more going on under the hood than just the number of pixels.

Yeah I'm getting 20fps reduction in Kena Bridge of Spirit with DLDSR + DLSS Quality combo compared to DLAA as well. 

How do you adjust the sharpening with DLAA btw? I don't see any lines in the dlsstweaks ini file to do that?

**Comment by TheAlpha31** [comment]
> How do you adjust the sharpening with DLAA btw? I don't see any lines in the dlsstweaks ini file to do that?

It's nothing DLAA or DLSS specific, you can just use the Sharpen slider under Image Scaling in the Nvidia Control Panel. It'll work even if you're running at native resolution. 
<details><summary>Screenshot</summary>
<p>

![image](https://user-images.githubusercontent.com/50899632/229992641-0ccf0771-e096-41ad-bf31-b4e55c4870bb.png)

</p>
</details> 

Death Stranding does have a sharpening option in-game, but I think it broke when I upgraded the DLSS version.

**Comment by ShinChaosBahamut** [comment]
> > How do you adjust the sharpening with DLAA btw? I don't see any lines in the dlsstweaks ini file to do that?
> 
> It's nothing DLAA or DLSS specific, you can just use the Sharpen slider under Image Scaling in the Nvidia Control Panel. It'll work even if you're running at native resolution.
> Screenshot
> 
> Death Stranding does have a sharpening option in-game, but I think it broke when I upgraded the DLSS version.

DLSS versions starting with 2.5 don't use any sharpening. So the in-game option did 'break', but that's intended behavior. nVidia's Image Scaling or a Reshade sharpener filter will do the job instead.

**Comment by warblgarbl** [comment]
Signs have some ghosting issues with lateral movement, especially on the C and D presets. C is worst, followed closely by D. F isn't too bad and is at least legible. A and B are the best at handling this, but those have far worse trailing on the floating particles.

[https://imgsli.com/MjE0NzE4/0/2](https://imgsli.com/MjE0NzE4/0/2)
